e2e_tests/tests/cluster/test_master_restart.py

```diff
@@ -4,33 +4,22 @@
 from typing import Iterator
 
 import pytest
-import requests
 
-from determined.common import constants
-from determined.common.api import authentication
 from determined.common.api.bindings import determinedexperimentv1State as EXP_STATE
-from tests import command as cmd
 from tests import config as conf
 from tests import experiment as exp
-from tests.cluster.test_users import det_spawn
 
 from .managed_cluster import ManagedCluster
-from .utils import (
-    command_succeeded,
-    get_command_info,
-    run_command,
-    wait_for_command_state,
-    wait_for_task_state,
-)
+from .utils import command_succeeded, run_command, wait_for_command_state
 
 logger = logging.getLogger(__name__)
 
 
-def _sanity_check(managed_cluster: ManagedCluster) -> None:
-    if not managed_cluster.reattach:
+def _sanity_check(managed_cluster_restarts: ManagedCluster) -> None:
+    if not managed_cluster_restarts.reattach:
         pytest.skip()
 
-    managed_cluster.ensure_agent_ok()
+    managed_cluster_restarts.ensure_agent_ok()
 
 
 @pytest.fixture
```

Comment:
`restartable_managed_cluster` in the next line here is using `managed_cluster` fixture
1.  it probably should use `managed_cluster_restarts` instead
2. `managed_cluster` fixture was removed, how come it still works